lib/msal-browser/src/controllers/ControllerFactory.ts

```diff
@@ -3,38 +3,36 @@
  * Licensed under the MIT License.
  */
 
-import { TeamsAppOperatingContext } from "../operatingcontext/TeamsAppOperatingContext";
+import { NestedAppOperatingContext } from "../operatingcontext/NestedAppOperatingContext";
 import { StandardOperatingContext } from "../operatingcontext/StandardOperatingContext";
 import { IController } from "./IController";
 import { Configuration } from "../config/Configuration";
+import { StandardController } from "./StandardController";
+import { NestedAppAuthController } from "./NestedAppAuthController";
 
 export async function createV3Controller(
     config: Configuration
 ): Promise<IController> {
     const standard = new StandardOperatingContext(config);
 
     await standard.initialize();
-
-    const controller = await import("./StandardController");
-    return controller.StandardController.createController(standard);
+    return StandardController.createController(standard);
 }
 
 export async function createController(
     config: Configuration
 ): Promise<IController | null> {
     const standard = new StandardOperatingContext(config);
-    const teamsApp = new TeamsAppOperatingContext(config);
+    const nestedApp = new NestedAppOperatingContext(config);
```

Comment:
this is just called from PCANext and could be removed when we drop PCANext, right? is it worth just getting rid of the NAA stuff from PCANext right now and slimming this down?

Reply:
Yes, added comment above. Will follow up with a PR to clean controller creation altogether.